freeipa.spec.in

```diff
@@ -313,8 +313,7 @@ Requires: python3-pyldap >= %{python3_ldap_version}
 Requires: python2-ipaserver = %{version}-%{release}
 Requires: python2-ldap >= %{python2_ldap_version}
 %endif
-# 1.3.7.6-1: https://bugzilla.redhat.com/show_bug.cgi?id=1488295
-Requires: 389-ds-base >= 1.3.7.6-1
+Requires: 389-ds-base >= 1.3.7.9-1
```

Comment:
Please include a comment why 1.3.7.9 is required.

Reply:
Can't folks just use `git blame` if they're curious?

Reply:
I'm fine with that, other devs aren't.

Reply:
Righto, I'll add a comment.

Reply:
The reasoning is due to different versions between distros so when doing a release and comparing spec files it requires a lot less dereferencing to figure out whether the required bug fix is in the available versions whose numbers may not always line up.